ControllerDriver/networkhandler.h

```diff
@@ -0,0 +1,27 @@
+#ifndef NETWORKHANDLER_H
+#define NETWORKHANDLER_H
+
+#include <Arduino.h>
+
+typedef struct Packet {
+	uint8_t* data;
+  uint32_t length;
+	uint8_t priority;
```

Comment:
It doesn't appear that `priority` is used anywhere yet.  I assume it'll be used in the future in `handleNetworkStuff`?

Reply:
Yeah. My thought was that sometimes packets will have to be dropped if running out of memory, so anything with priority 255 won't be dropped at all and anything lower is dropped in order of priority.